order-routing/javascript/pickup-point-delivery-option-generators/default/README.md

```diff
@@ -28,6 +28,7 @@ which contains pickup point information in the following format:
           }
         }
       },
+      // Note: "openingHours" is an optional field
```

Comment:
let's remove the notes. nullable can be seen in API doc in and in payload and in code